packages/vaadin-grid/test/column.test.js

```diff
@@ -552,4 +552,13 @@ describe('column', () => {
       expect(column._bodyTemplate).to.eql(template2);
     });
   });
+
+  it('should not throw an exception when size is changed after removing column', () => {
```

Comment:
If the title says "should not throw", there should be a check such as `expect(...).not.to.throw(Error);`

Reply:
Done